informer/integration/custom_resource.go

```diff
@@ -0,0 +1,101 @@
+// +build k8srequired
+
+package integration
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+const (
+	namespace = "informer-integration-test"
+)
+
+type CustomResource struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata"`
+	Spec              CustomResourceSpec `json:"spec"`
+}
```

Comment:
I don't think this type is needed, probably some copied dead code.

Reply:
Gone.